src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -187,6 +211,211 @@ class UserService(protected val userInfo: UserInfo, dataSource: DataSource, prot
     }
   }
 
+  def listAdmins() = {
+    asAdmin {
+      gcsDAO.listAdmins.map(RequestComplete(StatusCodes.OK, _)).recover{ case throwable =>
```

Comment:
On a related note, I should probably either close the other PR that does exactly this, or not do it in this PR since it seems that we need to discuss it as a group a bit more.